packages/visual-editor/src/components/puck/GetDirections.tsx

```diff
@@ -26,7 +26,8 @@ const getDirectionsFields: Fields<GetDirectionsProps> = {
     label: "Variant",
     type: "radio",
     options: [
-      { label: "Button", value: "primary" },
+      { label: "Primary", value: "primary" },
```

Comment:
Case in point, this is repeated again. Maybe not for this PR but we audit and do this for all of these type of theme options in one swoop.